ArchiSteamFarm/Bot.cs

```diff
@@ -1173,8 +1173,17 @@ public sealed class Bot : IDisposable {
 			// We must escape our message prior to sending it
 			message = Escape(message);
 
-			for (int i = 0; i < message.Length; i += maxMessageLength) {
-				string messagePart = message.Substring(i, Math.Min(maxMessageLength, message.Length - i));
+			int i = 0;
+			while (i < message.Length) {
+				int partLength;
+				if (message.Length - i > maxMessageLength) {
+					int lastNewLine = message.LastIndexOf(Environment.NewLine, i + maxMessageLength, maxMessageLength);
+					partLength = lastNewLine > i ? lastNewLine - i : maxMessageLength;
+				} else {
+					partLength = message.Length - i; //less than maxMessageLength left
```

Comment:
If you've decided to remove comments, then this one too. Otherwise, prefer comments in standalone lines (unless impossible or impractical due to specific code structure, but here totally valid).